### PR TITLE
Develop/2.6 edge Edge management, enableEdge

### PR DESCRIPTION
**Pull request by deaflynx**

### CODE REVIEW

File: ui/src/app/api/user.service.js [hunk 1]
```diff
@@ -65,7 +66,9 @@ function UserService($http, $q, $rootScope, adminService, dashboardService, time
         reloadUser: reloadUser,
         isUserTokenAccessEnabled: isUserTokenAccessEnabled,
         loginAsUser: loginAsUser,
-        setUserCredentialsEnabled: setUserCredentialsEnabled
+        setUserCredentialsEnabled: setUserCredentialsEnabled,
+        isEdgesSupportEnabled: isEdgesSupportEnabled,
+        loadIsEdgesSupportEnabled: loadIsEdgesSupportEnabled
```
Comment (volodymyr-babak) [inline ui/src/app/api/user.service.js:71]: why this method must be public?

File: ui/src/app/services/menu.service.js [hunk 2]
```diff
@@ -406,6 +417,11 @@ function Menu(userService, $state, $rootScope) {
                                 }
                             ]
                         }];
+
+                    if (userService.isEdgesSupportEnabled()) {
+                        sections.splice(4, 0, edgeMenuSection.side);
```
Comment (volodymyr-babak) [inline ui/src/app/services/menu.service.js:422]: let's avoid this splice method.
Please add edge sections and home sections to array if edge support enabled.

File: ui/src/app/services/menu.service.js [hunk 3]
```diff
@@ -316,7 +319,31 @@ function Menu(userService, $state, $rootScope) {
                                 ]
                             }];
 
+                    if (userService.isEdgesSupportEnabled()) {
+                        sections.splice(6, 0, edgeMenuSection.side);
```
Comment (volodymyr-babak) [inline ui/src/app/services/menu.service.js:323]: let's avoid this splice method.
Please add edge sections and home sections to array if edge support enabled.

File: ui/src/app/api/user.service.js (hunk 1 above)
Comment (deaflynx) [reply ui/src/app/api/user.service.js:71]: good point. removed loadIsEdgesSupportEnabled() from public scope

**Pull request merged**